executor/platform_compat/windows/permissions.py

```
@@ -0,0 +1,149 @@
+            try:
+                token = win32security.OpenProcessToken(
+                    win32api.GetCurrentProcess(), win32security.TOKEN_QUERY
+                )
+                self._current_user_sid = win32security.GetTokenInformation(
+                    token, win32security.TokenUser
+                )[0]
+            except Exception:
+                pass
+        return self._current_user_sid
+
+    def set_owner_only(self, path: str, is_directory: bool = False) -> None:
```

Ruff (0.14.14)

[warning] Unused method argument: `is_directory`

(ARG002)